content/pt/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html

```diff
@@ -0,0 +1,115 @@
+
+    <link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+
+<div class="layout" id="top">
+
+    <main class="content">
+
+        <div class="row">
+
+         <div class="col-md-8">
+          <h3>Objetivos</h3>
+                <ul>
+                    <li> Saiba mais sobre implantações de aplicativos. </li>
+                    <li> Implante seu primeiro aplicativo no Kubernetes com o kubectl. </li>
+                </ul>
+            </div>
+
+            <div class="col-md-8">
+                <h3>Implantações do Kubernetes</h3>
+                <p>
+                Assim que o seu cluster do Kubernetes estiver em execução você pode implementar seu aplicativo em contêiners nele.
+                Para fazer isso, você precisa criar uma configuração do tipo <b> Deployment </b> do Kubernetes. O Deployment define como criar e 
+                atualizar instâncias do seu aplicativo. Depois de criar um Deployment, o Master do Kubernetes
+                agenda as instâncias do aplicativo incluídas nesse Deployment para ser executado em nós individuais do CLuster.
+                </p>
+
+                <p> Depois que as instâncias do aplicativo são criadas, um Controlador do Kubernetes Deployment monitora continuamente essas instâncias. 
+                    Se o nó que hospeda uma instância ficar inativo ou for excluído, o controlador de Deployment substituirá a instância por uma instância em outro nó no cluster. 
+                <b> Isso fornece um mecanismo de autocorreção para lidar com falhas ou manutenção da máquina. </b> </p>
+
+                <p>Em um mundo de pré-orquestração, os scripts de instalação costumavam ser usados ​​para iniciar aplicativos, mas não permitiam a recuperação de falha da máquina. 
+                   Ao criar suas instâncias de aplicativo e mantê-las em execução entre nós, as implantações do Kubernetes fornecem uma abordagem fundamentalmente diferente para o gerenciamento de aplicativos. </p>
+
+            </div>
+
+            <div class="col-md-4">
+                <div class="content__box content__box_lined">
+                    <h3>Resumo:</h3>
+                    <ul>
+                        <li>Deployments</li>
+                        <li>Kubectl</li>
+                    </ul>
+                </div>
+                <div class="content__box content__box_fill">
+                    <p><i>
+                        O tipo Deployment é responsável por criar e atualizar instâncias de seu aplicativo
+                    </i></p>
+                </div>
+            </div>
+        </div>
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+                <h2 style="color: #3771e3;">Implantar seu primeiro aplicativo no Kubernetes</h2>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col-md-8">
+                <p><img src="/docs/tutorials/kubernetes-basics/public/images/module_02_first_app.svg"></p>
```

Comment:
Should this be `/pt/docs/tutorials/kubernetes-basics/public/images/module_02_first_app.svg` (I think so)

Reply:
I don't believe we have the images localized yet as well. Do you think we should add the public folder with the images event if we are not translating those?